test/test_descriptor.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert')
+const generateDescriptor = require('../lib/descriptor.js').generateDescriptor
+
+describe('generate dp', () => {
+  it('from inline data', function() {
+    const csvData = "a,b,c\n1,2,3\n4,5,6"
```

Comment:
can we have something else besides integers Eg: date, string.. so we could be sure that types for fields are set correctly